setup.json

```diff
@@ -29,7 +29,7 @@
   ],
   "reentry_register": true,
```

Comment:
Could it be that this is no longer needed?

Reply:
Indeed!